source/wp-content/themes/wporg-parent-2021/sass/blocks/_search.scss

```diff
@@ -63,24 +74,31 @@
 		background-color: var(--wp--custom--form--color--background);
 		font-size: var(--wp--custom--form--typography--font-size);
 		line-height: var(--wp--custom--button--typography--line-height);
+		color: var(--wp--custom--form--active--color--text);
 
 		&::placeholder {
 			opacity: 1;
-			color: var(--wp--custom--form--color--placeholder);
+			color: var(--wp--custom--form--color--text);
 		}
 
 		&::-webkit-search-cancel-button {
 			-webkit-appearance: none;
 			height: 10px;
 			width: 10px;
-			background: transparent;
-			background-image: url(../images/search-x.svg);
+			background-color: var(--wp--custom--form--active--color--text);
+			vertical-align: baseline;
+			mask-image: url(../images/search-x.svg?v=2);
+			mask-size: cover;
+		}
+
+		&:focus::-webkit-search-cancel-button {
+			background-color: var(--local--focus--outline-color);
```

Comment:
For me on Developer, this changes the color of the clear icon to white when the input is focused and it disappears. From the screenshots I expected it to be blue, in which case I think this might need to be:

```suggestion
			background-color: var(--local--focus--border-color);
```

![Developer search style](https://github.com/WordPress/wporg-parent-2021/assets/1017872/f807bb08-40a0-4a7b-b9bd-ae44cd9d0384)